### PR TITLE
use latest inter

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
[Issue](https://linear.app/electricitymaps/issue/ELE-1517/use-correct-variable-fonts-for-inter
)

## Description
Adds variable inter fonts from here https://github.com/rsms/inter/releases/tag/v3.19

### CODE REVIEW

File: web/src/index.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
   @font-face {
     font-family: Inter;
     font-weight: 400 700;
-    src: url('/fonts/Inter-Variable.woff2') format('woff2-variations');
+    src: url('/fonts/var.woff2') format('woff2-variations');
```
Comment (VIKTORVAV99) [inline web/src/index.css:9]:
```suggestion
    src: url('/fonts/Inter.var.woff2') format('woff2-variations');
```
This should be `Inter.var.woff2` right?

Comment (tonypls) [reply web/src/index.css]: Yes, good spotting. Updated with a bad pun commit message :D

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

Apparently I don't have the same powers in here so you will have to wait for another review 😅

**Comment by VIKTORVAV99** [comment]
Oh I missed this before but don't forget to update the preload in the index.html too.

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**